### ISSUE TITLE
Age-Restriction bypass not working anymore

### ISSUE THREAD

**Issue by WesleyTRV** [zerodytrash/Simple-YouTube-Age-Restriction-Bypass#247]
### What happened?

The script is not working anymore, either because of the Tampermonky update of the script itself.

### Platform

Userscript

### Version

v2.5.9

### What browser are you on?

Chrome

### Browser version

127.0.6533.89

### Link to the video

https://www.youtube.com/watch?v=fb1MR85XFOc

### Screenshots or Videos

![image](https://github.com/user-attachments/assets/2a10fb41-f395-4c34-b109-ac2a2407b0cb)

### About you

- [ ] I am a developer and/or have knowledge in programming

**Comment by drunkwinter** [comment]
Hi, does it work with Violentmonkey? Tampermonkey tends to break after updates, in which case toggling the extension on/off as a workaround will usually fix it.

**Comment by drunkwinter** [comment]
Okay, seems like a new browser security thing that's causing this...

Will fix.

**Comment by drunkwinter** [comment]
Now I cannot seem to reproduce the issue reliably anymore. If any of you can provide us with logs from the developer console it would help allot. Press F12 and filter for "Simple-YouTube-Age-Restriction-Bypass" after you notice the behavior.

**Comment by Snipercupcakes** [comment]
hi, im having the same issue here. 
here are my console logs:

Uncaught (in promise) TypeError: Failed to set the 'innerHTML' property on 'Element': This document requires 'TrustedHTML' assignment.
    at Function.assign (<anonymous>)
    at createElement (userscript.html?name=Simple-YouTube-Age-Restriction-Bypass.user.js&id=92bafaae-a99f-4e26-b17c-f3a64849f423:126:27)
    at iife (userscript.html?name=Simple-YouTube-Age-Restriction-Bypass.user.js&id=92bafaae-a99f-4e26-b17c-f3a64849f423:867:29)
    at Object.<anonymous> (userscript.html?name=Simple-YouTube-Age-Restriction-Bypass.user.js&id=92bafaae-a99f-4e26-b17c-f3a64849f423:1356:3)
    at wt (<anonymous>:9:89)
    at userscript.html?name=Simple-YouTube-Age-Restriction-Bypass.user.js&id=92bafaae-a99f-4e26-b17c-f3a64849f423:1:90
    at window.__f__lzcq9538.meu (userscript.html?name=Simple-YouTube-Age-Restriction-Bypass.user.js&id=92bafaae-a99f-4e26-b17c-f3a64849f423:1:341)
    at wt (<anonymous>:9:89)
    at o (<anonymous>:80:114)
    at <anonymous>:81:275

**Comment by Dusty211** [comment]
Not working for me as well.  

- Simple YouTube Age Restriction Bypass v2.5.9
- Tampermonkey Chrome extension v5.2.3
- Linux
- Chrome is up to date
    - Version 127.0.6533.89 (Official Build) (64-bit)
    - Incognito mode
    - uBlock origin enabled
YouTube Links:
- https://www.youtube.com/watch?v=fb1MR85XFOc
- https://www.youtube.com/watch?v=z0NfI2NeDHI

Happy to provide other useful information.  Thank you for the software.

**Comment by drunkwinter** [comment]
I'll have a build ready later today for you guys to test. 

@Dusty211 If there are no logs, I'm afraid the userscript is not running at all. Check if it is toggled on in Tampermonkey. If it is, try toggling Tampermonkey on/off. If that does not work either try it with Violentmonkey.

**Comment by drunkwinter** [comment]
Please let me know of any of these builds work. I think YouTube is A/B testing the new `trustedTypes` browser feature, so perhaps the issue might've just disappeared for many like it did with me, so please check if you still have the issue.

**Instruction:** Copy the file contents and paste it into a new or existing `Simple-YouTube-Age-Restriction-Bypass` userscript.
- [Build 1](https://github.com/user-attachments/files/16475681/Simple-YouTube-Age-Restriction-Bypass.user.js.txt)
- [Build 2](https://github.com/user-attachments/files/16475670/Simple-YouTube-Age-Restriction-Bypass.user.js.txt)

**Comment by fokori7216atmvpalacedotcom** [comment]
@drunkwinter Build one did not fix it for for me, but build 2 did.

My version of chrome is currently one update behind though, so that is worth mentioning, SPL might stop working after I relaunch to update. 

Also just in case this is matters, I am using Violent Monkey, incognito mode with ublock origin also running.

 Thanks!

**Comment by drunkwinter** [comment]
Alright, waiting for more feedback before pushing a release.

**Comment by LFL38** [comment]
> Please let me know if any of these builds work. I think YouTube is A/B testing the new `trustedTypes` browser feature, so perhaps the issue might've just disappeared for many like it did with me, so please check if you still have the issue.
> 
> **Instruction:** Copy the file contents and paste it into a new or existing `Simple-YouTube-Age-Restriction-Bypass` userscript.
> 
> * [Build 1](https://github.com/user-attachments/files/16475681/Simple-YouTube-Age-Restriction-Bypass.user.js.txt)
> * [Build 2](https://github.com/user-attachments/files/16475670/Simple-YouTube-Age-Restriction-Bypass.user.js.txt)

Thanks so much man!! (Build 2 works, didn't test Build 1)

**Comment by drunkwinter** [comment]
Thanks to everyone who provided any feedback.

Fixed in [v2.5.10](https://github.com/zerodytrash/Simple-YouTube-Age-Restriction-Bypass/releases/tag/v2.5.10)

**Comment by Dusty211** [comment]
@drunkwinter Sorry for the late response; I was out of town. Thank you, you were right.  It wasn't even running, and I came here and saw this and thought it was affecting me, but the real problem was that I didn't have TamperMonkey configured properly.  I needed to enable "developer mode" in the Chrome extensions dashboard, and now everything works great using your bypass v2.5.10.  Sorry for the confusion, and again, thanks for the software.
